modules/AOR_Reports/aor_utils.php

```diff
@@ -570,3 +585,19 @@ function convertToDateTime($value)
 
     return $dateTime;
 }
+
+function setMinTime(DateTime $dt){
+    if (version_compare(PHP_VERSION, '7.1.0') >= 0) {
+        $dt->setTime(0, 0, 0, 0);
+    } else {
+        $dt->setTime(0, 0, 0);
+    }
+}
```

Comment:
Can this be dropped? I don't see it being used anywhere.

Reply:
I have a tenancy to like having the inverse method to setMaxTime.
Setting microtime is just to make sure we are always at a known microtime of 0, as say now() may return microseconds above 0 and you will have DB issues when doing >=0 vs >=0.124